backend/owa.ts

```diff
@@ -1,8 +1,21 @@
 import { session as Session, BrowserWindow } from "electron";
 
+const kCookieName = "X-OWA-CANARY";
+const kHotmailServer = "outlook.live.com";
+const reOWAPath = /^\/owa\/(\d+\/){0,1}/;
```

Comment:
Owl has:
```
          details.cookie.domain == kHotmailServer &&
          this.cookieNames.includes(details.cookie.name) &&
          details.cookie.path.startsWith("/owa/")) {
```
Can you please use `.startsWith("/owa/")` instead of the regexp?